### PR TITLE
Create initial structure for Tanzu Observability exporter

### PR DESCRIPTION
**Pull request by thepeterstone**
Splitting #3392

**Description:**  This is the first PR which creates our README, config and no-op factory.

**Testing:** 
Unit tests

### CODE REVIEW

File: exporter/tanzuobservabilityexporter/README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+receivers:
+  examplereceiver:
+
+processors:
+  batch:
+    timeout: 10s
+
+exporters:
+  tanzuobservability:
+    traces:
```
Comment (bogdandrutu) [inline exporter/tanzuobservabilityexporter/README.md:37]: Are you planning to have support for other signals?

File: exporter/tanzuobservabilityexporter/config.go [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+import (
+	"fmt"
+
+	"go.opentelemetry.io/collector/config"
+)
+
+type TracesConfig struct {
+	// Wavefront Proxy URL of the form protocol://host[:port]
+	Endpoint string `mapstructure:"endpoint"`
```
Comment (bogdandrutu) [inline exporter/tanzuobservabilityexporter/config.go:25]: What is the protocol? HTTP? If yes we have a `confighttp` helper to keep everything synchronize.

File: exporter/tanzuobservabilityexporter/config.go [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+	// A default value to use if a span doesn't contain an "application" label
+	DefaultApplication string `mapstructure:"default_application"`
+
+	// A default value to use if a span doesn't contain a "service" or "service.name" label
```
Comment (bogdandrutu) [inline exporter/tanzuobservabilityexporter/config.go:30]: The span does not have a `service.name` it is in the `Resource`. I would not add this initially, users can use the `attributeprocessor` to do these kind of changes.

File: exporter/tanzuobservabilityexporter/README.md (hunk 1 above)
Comment (oppegard) [reply exporter/tanzuobservabilityexporter/README.md:37]: Yes, we plan to support metrics in the future.

File: exporter/tanzuobservabilityexporter/config.go (hunk 2 above)
Comment (oppegard) [reply exporter/tanzuobservabilityexporter/config.go:25]: Yes, it's HTTP and we'll look at using `confighttp` in an update to this PR.

File: exporter/tanzuobservabilityexporter/config.go (hunk 3 above)
Comment (oppegard) [reply exporter/tanzuobservabilityexporter/config.go:30]: Thanks for pointing us to the `attributeprocessor`. We'll remove `DefaultService` and `DefaultApplication` from our exporter config and point users to `attributeprocessor` if they want to override those values.

Comment (bogdandrutu) [reply exporter/tanzuobservabilityexporter/config.go:30]: So are you going to remove this?

Comment (thepeterstone) [reply exporter/tanzuobservabilityexporter/config.go]: Yes - removed in aa2ffe.

File: exporter/tanzuobservabilityexporter/config.go (hunk 2 above)
Comment (thepeterstone) [reply exporter/tanzuobservabilityexporter/config.go]: Fixed to use `confighttp` in d5ef662.

**Review by bogdandrutu — APPROVED** [review]

**Pull request merged**